application_form/models.py

```diff
@@ -98,3 +102,55 @@ class ApplicationApartment(models.Model):
         default=ApplicationState.SUBMITTED,
         verbose_name=_("application state"),
     )
+
+
+class ReservationQueue(models.Model):
```

Comment:
The name of this model could be better if it is `HousingQueue` (Asuntojono). The reservation state is one of the states in the item.